src/mad_spark_alt/core/multi_perspective_prompts.py

```
@@ -0,0 +1,233 @@
+"""
+Multi-Perspective QADI Prompts
+
+This module provides perspective-specific prompts for the QADI methodology,
+enabling analysis from environmental, personal, technical, business, scientific,
+and philosophical viewpoints.
+"""
+
+from typing import Dict
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)